.gitignore

```diff
@@ -3,6 +3,7 @@
 
 # Ignore Node Modules
 node_modules/
+package-lock.json
```

Comment:
As per the [npm documentation](https://docs.npmjs.com/cli/v7/configuring-npm/package-locks#using-locked-packages)

> It is highly recommended you commit the generated package lock to source control: this will allow anyone else on your team, your deployments, your CI/continuous integration, and anyone else who runs npm install in your package source to get the exact same dependency tree that you were developing on. Additionally, the diffs from these changes are human-readable and will inform you of any changes npm has made to your node_modules, so you can notice if any transitive dependencies were updated, hoisted, etc.